mne/channels/montage.py

```diff
@@ -1634,3 +1646,38 @@ def make_standard_montage(kind):
                                         standard_montage_look_up_table.keys()))
     else:
         return standard_montage_look_up_table[kind]()
+
+
+def read_dig_eeglab(fname):
+    """Read an EEGLAB digitization file.
+
+    Parameters
+    ----------
+    fname : str
+        The filepath of Digitization EEGLAB formatted file.
+        File extension is expected to be '.loc', '.locs' or '.eloc'.
+
+    Returns
+    -------
+    montage : instance of DigMontage
+        The montage.
+
+    See Also
+    --------
+    make_dig_montage
+    """
+    VALID_FILE_EXT = ('.loc', '.locs', '.eloc')
+
+    _, ext = op.splitext(fname)
+    _check_option('fname', ext, VALID_FILE_EXT)
+    _check_fname(fname, overwrite='read', must_exist=True)
+
+    ch_names = np.genfromtxt(fname, dtype=str, usecols=3).tolist()
+    topo = np.loadtxt(fname, dtype=float, usecols=[1, 2])
+    sph = _topo_to_sph(topo)
+    pos = _sph_to_cart(sph)
+    pos[:, [0, 1]] = pos[:, [1, 0]] * [-1, 1]
+
+    # XXX: This is not a valid DigMontage. Whatever we are parsing is mapped
+    #      to radius 1 sphere
```

Comment:
why don't you map it to 0.85 radius like for the others?

Reply:
I see, what I meant is that this `read_dig_eeglab` is not of the same kind of `read_dig_captrack` x, y, z from an arbitrary coord ref. It only parses 2 angles. So you can only recover some sort of 'custom' template.